src/test/java/org/fcrepo/camel/integration/FcrepoConstantsIT.java

```diff
@@ -47,5 +47,8 @@ public void testRdfLexicon() {
         assertEquals(RdfNamespaces.INDEXING, RdfLexicon.INDEXING_NAMESPACE);
         assertEquals(RdfNamespaces.RDF, RdfLexicon.RDF_NAMESPACE);
         assertEquals(RdfNamespaces.LDP, RdfLexicon.LDP_NAMESPACE);
+        assertEquals(RdfNamespaces.REPOSITORY + "ServerManaged", RdfLexicon.SERVER_MANAGED.toString());
```

Comment:
The point of this test was to ensure that the locally defined constants match the constants defined in the fcrepo codebase. That way, if any `org.fcrepo.kernel.RdfLexicon` constants were to change, the test would fail.